setup.php

```diff
@@ -26,7 +26,7 @@
  --------------------------------------------------------------------------
  */
 
-define ('PLUGIN_TAG_VERSION', '2.7.0');
+define ('PLUGIN_TAG_VERSION', '2.8.0');
```

Comment:
Please do not change it on your PR, it may lead to some conflicts with other PR. We do it when creating release.
```suggestion
define ('PLUGIN_TAG_VERSION', '2.7.0');
```